### PR TITLE
Added test for check connectivity via key pair

### PR DESCRIPTION
**Pull request by ntymtsiv**

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by roman-verchikov** [comment]
```Warning: Identity file devlab/.vagrant/machines/icehouse/virtualbox/private_key not accessible: No such file or directory.
Permission denied, please try again.```

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by roman-verchikov** [comment]
Looks good, functional tests succeed. Need one more reviewer

File: devlab/tests/config.py [hunk 1]
```diff
@@ -169,3 +153,55 @@
 GLANCE_CLIENT_VERSION = '1'
 NEUTRON_CLIENT_VERSION = '2.0'
 CINDER_CLIENT_VERSION = '1'
+
+
+# Keypairs to create/delete
+# Connected to user's list
+keypairs = [
```
Comment (realsystem) [inline devlab/tests/config.py]: Why do we need to hardcode things like this? As I understand we can generate keys when we need. Or do you need this keys as a pattern for assertion in test?

**Pull request merged**